quimb/tensor/tensor_core.py

```diff
@@ -1356,6 +1356,70 @@ def fuse(self, fuse_map, inplace=False):
 
     fuse_ = functools.partialmethod(fuse, inplace=True)
 
+    def unfuse(self, unfuse_map, shape_map, inplace=False):
+        """Reshape single indices into groups of multiple.
+
+        Parameters
+        ----------
+        fuse_map : dict_like or sequence of tuples.
+            Mapping like: ``{existing_ind: sequence of new inds, ...}`` or an
+            ordered mapping like ``[(old_ind_1, new_inds_1), ...]`` in which
+            case the output tensor's new inds will be ordered. In both cases
+            the new indices are created at the old index's position of the 
+            tensor's shape.
+
```

Comment:
I think should be `unfuse_map` (and ideally no space between parameters in the docstring)